src/app/tide/spectrum_preprocess.h

```diff
@@ -117,19 +117,41 @@ class Spectrum;
 class ObservedPeakSet {
  public:
 
+  // This constructor is to be removed when marging TideSearchLite into the main branch.
```

Comment:
I don't understand the purpose here

Reply:
Sorry, this is my mistake, I should have deleted this code.